tecken/tests/test_api.py

```diff
@@ -1093,19 +957,59 @@ def test_stats(client):
     response = client.get(url)
     assert response.status_code == 200
     data = response.json()
-    assert data["stats"]["uploads"]
-    assert not data["stats"]["uploads"]["all_uploads"]
-    assert "users" not in data["stats"]
-    assert data["stats"]["tokens"]
+    assert data["stats"] == {
```

Comment:
I adjusted this test so it's clearer wtf `data["stats"]` has in it in the different modes.